task06/controller/error.controller.ts

```diff
@@ -0,0 +1,21 @@
+import { NextFunction, Request, Response } from 'express';
+import { AppError } from '../errors/appError';
+import { HttpStatusCode } from '../errors/statusCode';
+
+export const handleError = (
```

Comment:
Additionally, take into consideration name of the function, if you will move it to middleswares, it's better to have `*Middleware` within the name.

Reply:
Fixed